msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -425,7 +426,66 @@ else { /* handle case where capability isn't supported */ }
 
 ```
 
-#### Generate a deep link to a call
+## Generate a deep link to share content to stage in meetings
+
+You can also generate a deep link to [share the app to stage](~/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md#share-entire-app-to-stage) and start or join a meeting.
+
+> [!Note]
+> Deep link to share content to stage in meeting is supported in Teams desktop client only.
+
+When a deep link is selected in an app and the user is already in a meeting, the app is shared to stage, and a permission pop-up window appears. Users can grant permissions for the participants, such as co-editing a document or collaborating with an app.
+
+:::image type="content" source="../../assets/images/intergrate-with-teams/screenshot-of-pop-up-permission.png" alt-text="The screenshot is an example that shows a permission pop-up window.":::
+
+> [!NOTE]
+> Permission pop-up window appears only for Office apps.
+
+When user isn't in a meeting then user is redirected to the Teams calendar where user needs to join a meeting or an instant meeting (Meet now) can be initiated.
+
+:::image type="content" source="../../assets/images/intergrate-with-teams/Screenshot-of-meet-now-join-pop-up.png" alt-text="The screenshot is an example that shows an option to join the meeting.":::
+
+Once the user joins the meeting, they can add participants and interact with the app.
+
+:::image type="content" source="../../assets/images/intergrate-with-teams/Screenshot-ofmeet-now-option-pop-up.png" alt-text="The screenshot is an example that shows an option to add participants and how to interact with the app.":::
+
+To add a deep link to share content on stage, you need to have a app context, as in the following example.
+
+* `{ "appSharingUrl" : "https://teams.microsoft.com/extensibility-apps/meetingapis/view", "appId": "9ec80a73-1d41-4bcb-8190-4b9eA9e29fbb" , “useMeetNow”: false}`
+
+The query parameters are:
+
+* `appContext`: This consists the following
+
+  * `appID`: It allows the Teams client to fetch the app manifest and check if the sharing on stage is possible. It can be taken from the app manifest or it can be the ID that is used to register an app in the Teams app store.
+  * `appSharingUrl`: The URL which needs to be shared on stage.
+  * `useMeetNow`: It allows to create a Meet now meeting while using deep link for sharing on stage. The boolean parameters are either true or false. The default value is false, meaning if there’s no meeting, the calendar popup will be shown.
```

Comment:
> It allows to create a Meet now meeting

Can you clarify what you mean by this, as "allows" is ambiguous. Does this mean that the user gets the option to *choose* whether to create an instant meeting, vs a scheduled meeting? Or when useMeetNow=true, an instant meeting is automatically created?